src/tests/test_pyproxy.py

```diff
@@ -62,6 +62,49 @@ def get_value(self, value):
     assert selenium.run_js("return pyodide.pyimport('f').toString()").startswith("<Foo")
 
 
+def test_pyproxy2(selenium):
```

Comment:
The test name needs to be meaningful, here and below

Reply:
Yes of course =) I swear I meant to rename them. When I first name them, I'm thinking more about whether `pytest -k whatever` will get them and the test I already had.

This test is actually just a redundant minimization of an existing test, so either this should be removed or the other test should be broken up into smaller pieces.